### ISSUE TITLE
No RMSE decreasing? No improvements? Is it possible?

### ISSUE THREAD

**Issue by Temmplar** [dnouri/nolearn#127]
I started to use nolearn convolution net. I have 170K images after some preprocessing.
I configured my net, and ran it, but I didn't get the result I need.
This a regression task.
On training set I got decreasing RMSE starting from 0,25, but on validation set I experienced  almost no RMSE decreasing,
but rather it increases it almost from the beginning.
What would be the problem? I mean of course I know, that there is a lot option I can choose at configuring the net,
but there would be some basic problem somewhere in my process, which can produce this issue?
I would accept if the training process provided some even tiny positive validation result and then worsening, but how can it be that I can get any improvement?
Thank you in advance!
George

**Comment by dnouri** [comment]
Please don't crosspost this to several channels; it's rude.

**Comment by Temmplar** [comment]
Dear Daniel,

Firstly sorry for crossposting, but first I tried on lasagne groups, then I realized, that there is a more competent forum here, that's why this manner.
Furthermore this wasn't rude, rather desperate you can believe it and I believed that the best answer will be provided by you, because I watch continuously and with high respect your activity and blogs and anything you wrote so far. I'm working with your great code with your mind with your "spirituality", and I am quite desperate now and I was looking forward to your kind and mind-blowing answer.
Have a nice day!

Best regards,
George

**Comment by dnouri** [comment]
According to timestamps, it took you four minutes before you "realized, that there is a more competent forum here" and crosspost between the [Lasange mailing list](https://groups.google.com/forum/#!topic/lasagne-users/Vtudb1A2KQY) and here.  Sorry, but this looks like nothing more than an excuse.
